### PR TITLE
Add RateLimiter.Observe to some missing paths

### PR DESCRIPTION
**Pull request by alexkats**
* Add `RateLimiter.Observe` to some missing paths
* Make `RateLimiter` tests parallel

### CODE REVIEW

File: pkg/cloud/gen/main.go [hunk 1]
```diff
@@ -985,6 +985,8 @@ func (g *{{.GCEWrapType}}) AggregatedList(ctx context.Context, fl *filter.F) (ma
 		klog.V(4).Infof("{{.GCEWrapType}}.AggregatedList(%v, %v) = %v, %v", ctx, fl, nil, err)
 		return nil, err
 	}
+	g.s.RateLimiter.Observe(ctx, nil, ck)
```
Comment (bowei) [inline pkg/cloud/gen/main.go:988]: Is this path missing callObserverEnd as well?

File: pkg/cloud/gen/main.go [hunk 2]
```diff
@@ -1032,6 +1034,7 @@ func (g *{{.GCEWrapType}}) ListUsable(ctx context.Context, fl *filter.F) ([]*{{.
 		klog.V(4).Infof("{{.GCEWrapType}}.ListUsable(%v, ..., %v) = %v, %v", ctx, fl, nil, err)
 		return nil, err
 	}
+	g.s.RateLimiter.Observe(ctx, nil, ck)
```
Comment (bowei) [inline pkg/cloud/gen/main.go:1037]: callObserverEnd?

File: pkg/cloud/gen/main.go (hunk 1 above)
Comment (alexkats) [reply pkg/cloud/gen/main.go:989]: Right, thanks for pointing out, fixed

**Pull request merged**